src/components/molecules/UserProfilePicture/UserProfilePicture.scss

```diff
@@ -1,83 +1,152 @@
 @import "scss/constants.scss";
 
 $fadedWhite: rgba(255, 255, 255, 0.8);
-$borderWidth: 5px;
 
-.profile-picture-container {
-  position: relative;
+// TODO: this was copied from Audience.scss (do we want to move the variable to constants?)
+$avatarSize: max(4vh, 3vw);
```

Comment:
[TODO] this was copied from Audience.scss (do we want to move the variable to constants?)

Reply:
Yes, I would say that moving that to constant make sense, since we already reuse it in at least 2 places. (will be 3 with new auditorium)

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/3c6adcbfde6c7fc244a209afeac7de7460f79a05